server/src/db/migrations/20210802170819_createOwnedIngredients.cjs

```diff
@@ -0,0 +1,26 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.createTable("ownedIngredients", (table) => {
+    table.bigIncrements("id")
+    table.string("name").notNullable()
+    table.integer("measurement").notNullable()
+    table.string("unit").notNullable()
+    table.text("description")
+    table.bigInteger("userId").notNullable().unsigned().references("users.id")
+    table.timestamp("createdAt").notNullable().defaultTo(knex.fn.now())
+    table.timestamp("updatedAt").notNullable().defaultTo(knex.fn.now())
+  })
+}
```

Comment:
this table name is fine, though Im curious if an ingredient will ever have multiple owners (two people both have butter in the pantry). Im curious to poke around in a 1on1